onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -4295,7 +4309,7 @@ def aten_slice(
 ) -> TensorType:
     # slice.Tensor(Tensor(a) self, int dim=0, SymInt? start=None, SymInt? end=None, SymInt step=1) -> Tensor(a)
 
-    raise NotImplementedError()
+    return op.Slice(self, start, end, dim, step)  # type: ignore[arg-type]
```

Comment:
Not sure about this. This looks like it will fail in several cases. Are start/end/step supposed to be runtime values here? I assume so, based on the types. But handling the optional nature may be a bit tricky. Also, if specified, are they expected to be scalars or rank-1 tensors?